artifacts/publish_packages_finder.py

```diff
@@ -27,7 +27,6 @@
 import re
```

Comment:
Removed most of the methods which are not required, now it is simple.
1. List all the files.
2. Exclude packages that matched regex pattern provided under `exclude` config under artifact section in release config
3. Move final packages to dist directory to upload github artifactory

Reply:
Nice!